lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent_prompt.json

```diff
@@ -1,10 +0,0 @@
-{
-  "_type": "prompt",
-  "template": "Today is {date} and you can use tools to get new information. Answer the following questions as best you can using the following tools:\n\n{tools}\n\nUse the following format:\n\nQuestion: the input question you must answer\nThought: you should always think about what to do\nAction: the action to take, should be one of {tool_names}\nAction Input: the input to the action\nObservation: the result of the action\n... (this Thought/Action/Action Input/Observation can repeat N times)\nThought: I now know the final answer\nFinal Answer: the final answer to the original input question\n\nBegin!\n\nQuestion: {question}\nThought:",
```

Comment:
Just curious why'd we remote the final "Thought:"?

Reply:
I thought that demonstrates to the ai how to format the final thought as "Final Thought:", but I can try taking that out.

Reply:
Yeah, the scope of the ticket was to just convert the prompt from JSON to YAML. 

Thanks for being flexible!

Reply:
@andreibondarev Sure, but note that I don't think I changed the prompt for ChainOfThought.  I just moved it to yaml.

But I did just cleanup SQLQueryAgent prompting slightly and tested it.